spec.bs

```diff
@@ -3792,6 +3832,30 @@ step "Set [=this=]'s [=Request/request=] to |request|":
 
 </div>
 
+<div algorithm="iframe navigation capture adAuctionHeaders attribute patch">
+Modify the <{iframe}> element to add a
+<dfn element-attr for="iframe">adauctionheaders</dfn> <a spec=html>content attribute</a>.
+The IDL attribute {{HTMLIFrameElement/adAuctionHeaders}} [=reflects=]
+the <{iframe/adauctionheaders}> <a spec=html>content attribute</a>.
+
+<pre class=idl>
+partial interface HTMLIFrameElement {
+  [CEReactions] attribute boolean adAuctionHeaders;
+};
+</pre>
+</div>
+
+<div algorithm="iframe navigation patch">
+The following step will be added to the
+<a spec="html">create navigation params by fetching</a> steps
+after step "Let |request| be a new [=Request/request=], with ...":

```

Comment:
This doesn't really matter much since I'm commenting on the monkey patch, but can `[=Request/request=]` be `[=request=]`, or `<a for=/>request</a>`?

Reply:
It seems to point to something different: https://fetch.spec.whatwg.org/#concept-request-request (for `[=Request/request=]`) vs https://fetch.spec.whatwg.org/#concept-request (for `[=request=]`), so have to leave this as is. Thanks!

Reply:
Well we want to point to https://fetch.spec.whatwg.org/#concept-request, not https://fetch.spec.whatwg.org/#concept-request-request, so I think we should probably change this in a small follow-up PR.

Reply:
Ah, sorry, I got confused as to which line in the referenced spec we were matching. I had convinced myself that we were referencing step 28 of the Request constructor in the Fetch Standard ("Set [this](https://webidl.spec.whatwg.org/#this)’s [request](https://fetch.spec.whatwg.org/#concept-request-request) to request."), which points to #concept-request-request, instead of the line we're actually referencing - step 3 of the "create navigation params by fetching" in the HTML standard ("Let request be a new [request](https://fetch.spec.whatwg.org/#concept-request)..."

I'll send  a follow-up PR to fix this. I based this part on [the equivalent section in the Topics API draft spec](https://github.com/patcg-individual-drafts/topics/blob/main/spec.bs#L517). I'll follow up with them to recommend that change there as well.

Thanks!

Reply:
Created https://github.com/WICG/turtledove/pull/964. Thanks again!